src/Microsoft.Extensions.Logging.Console/ConfigurationConsoleLoggerSettings.cs

```diff
@@ -62,7 +62,7 @@ public bool TryGetSwitch(string name, out LogLevel level)
                 level = LogLevel.None;
                 return false;
             }
-            else if (Enum.TryParse<LogLevel>(value, out level))
+            else if (Enum.TryParse<LogLevel>(value, true, out level))
```

Comment:
Could you use a named param here - `Enum.TryParse<LogLevel>(value, ignoreCase: true, result: out level)`?